assets/app/view/game/buy_company_from_other_player.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module View
+  module Game
+    class BuyCompanyFromOtherPlayer < Snabberb::Component
+      include Actionable
+
+      needs :game, store: true
```

Comment:
do not list this as store since you're passing it in

Reply:
Fixed